Resources/Prototypes/Catalog/Cargo/cargo_livestock.yml

```diff
@@ -197,3 +197,13 @@
   cost: 2800
   category: Livestock
   group: market
+
+- type: cargoProduct
+  id: LivestockMothroach
+  icon: mothroach.rsi
```

Comment:
remove mothroach.rsi after `icon:`

Reply:
yeah, he's right. You gotta just leave it as `icon:`
aside from that, it should be good.